app/components/createService/includes/secret.html

```diff
@@ -1,7 +1,8 @@
 <form class="form-horizontal" style="margin-top: 15px;">
   <div class="form-group">
     <div class="col-sm-12 small text-muted">
-      Secrets will be available under <code>/run/secrets/$SECRET_NAME</code> in containers.
+      Secrets will be available under <code>/run/secrets/$SECRET_NAME</code> in containers.<br/>
+      You can override the default target file.
```

Comment:
This should only be shown in API version >= 1.30.

Reply:
Oh yeah good catch, I'm on it